.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: java
+
+jdk: 
+- oraclejdk8
```

Comment:
we work with Java8? :)

Reply:
Yes, I have setup travis-ci for now on my [Fork](https://travis-ci.org/kajdreef/blinky) to test it all out. When compiling locally java9 doesn't work, but I haven't looked into that yet. :)